meshio/gmsh/_gmsh41.py

```diff
@@ -155,13 +184,27 @@ def _read_nodes(f, is_ascii, data_size):
         # Store the point densely and in the order in which they appear in the file.
         # x(double) y(double) z(double) (* numNodes)
         points[ixx] = fromfile(f, c_double, num_nodes * 3).reshape((num_nodes, 3))
+
+        # Entity tag and entity dimension of the nodes. Stored as point-data.
+        entity_of_nodes[ixx] = entity_tag
+        entity_dimension[ixx] = dim
```

Comment:
Isn't this always 0?

Reply:
See comment above regarding entities in general.

Reply:
Gmsh calls this `dim_tag`, let's follow suit with a `dim_tag` array, not separate `entity_of_nodes` and `entity_dimension`.

Reply:
Agreed, this would be cleaner. However, storage as point data is then not permissible, since 
gmsh only accepts 1, 3 or 9 components per data fields, cf `meshio.gmsh.common._write_data()` L244. Should I store it in Mesh.point_sets instead? 

Sorry for the trouble; it is a bit hard to understand what is the intended usage for the different kinds of variables, although I appreciate the flexibility that comes with not specifying this clearly.

Reply:
Well, how do you write `entity_of_nodes` and `entity_dimension` now?

Reply:
They are two different arrays (because of the different keys) with 1 component each. Thus no problem